source/our_tools/flows.rst

```diff
@@ -0,0 +1,252 @@
+
+#. Also, you can download the singularity "container_scilus_1.6.0.sif" necessary for this test with this command:
+
+    .. code-block:: bash
+
+        curl https://nextcloud.computecanada.ca/index.php/s/NijdKTP7WWbP7Na/download -o containers_scilus_1.6.0.sif
+
+#. Finally, create the different directions for the different flows according to the structure of the inputs required for each.
+
+    .. code-block:: bash
+
+        for i in data_tuto_1.6.0/sub-*; do mkdir -p tractoflow_test/raw/$(basename $i);done
+        for i in data_tuto_1.6.0/sub-*; do mkdir -p RBx_flow_test/raw/$(basename $i);done
+        for i in data_tuto_1.6.0/sub-*; do mkdir -p tractometry_flow_test/raw/$(basename $i); done
+
+Tractoflow
+**********
+
+#. Add the data needed to launch tractoflow from downloaded data.
+
+    .. code-block:: bash
+
+        for i in data_tuto_1.6.0/sub-*; do cp ${i}/* tractoflow_test/raw/$(basename $i)/; done
+
+.. note::
+
+    The data are composed of 3 subjects and each subject contains 7 files: aparc+aseg.nii.gz, bval, bvec, dwi.nii.gz, rev_b0.nii.gz, t1.nii.gz, wmparc.nii.gz.
+    By default only bval, bvec, dwi.nii.gz, t1.nii.gz are necessary to run tractoflow. 
+    But here we're using tractoflow ABS, so aparc+aseg.nii.gz, rev_b0.nii.gz, and wmparc.nii.gz are required.
+
+#. Run tractoflow (this can take a long time).
+
+    .. code-block:: bash
+
+        nextflow ${FLOW_DIR}/tractoflow/main.nf --input tractoflow_test/raw --local_nbr_seeds 1 --run_eddy False \
+         --run_topup False --output_dir tractoflow_test/results_tf -profile ABS -with-singularity ./containers_scilus_1.6.0.sif \
+         -with-report tractoflow_test/report.html -w tractoflow_test/work -resume
+
+ Parameters:
+  - :bash:`--input`: direction of our data
+  - :bash:`--local_nbr_seeds`: number of seeds related to the seeding type param
+  - :bash:`--run_eddy`: activate or not eddy
+  - :bash:`--run_topup`: activate or not topup
+  - :bash:`--output_dir`: direction where results will be generate
+  - :bash:`-profile ABS`: choose the profile TractoFlow-ABS (Atlas Based Segmentation)
+  - :bash:`-with-singularity`: direction of singularity we want to use
+  - :bash:`-with-report`: generate a report a the end of the command
+  - :bash:`-w`: direction where work will be generate
+  - :bash:`-resume`: use the results already created in the work if the command has already been executed
+
+Here, tractoflow is set to be as fast as possible if you want to check more options run the command :
+
+    .. code-block:: bash
+        
+        nextflow ${FLOW_DIR}/tractoflow/main.nf --help
+
+Or you can check the documentation from tractoflow documentation `here <https://tractoflow-documentation.readthedocs.io/en/latest/pipeline/options.html>`_.
+    
+.. warning::
+    Once tractoflow is launched, a large number of files are created. Be careful, the files in results are only symlinks to the work. Do not delete your work!
```

Comment:
files in the results folder (--output_dir) are only symlinks to the "work" folder created by nextflow.